index.html

```diff
@@ -110,10 +112,11 @@ <h2>CSS Topics</h2>
   
                   It uses the "link" tag on every pages and the "link" tag should be put inside the head section.<br>
               </p></li>
+            </ul>
               <code>
                 link rel="stylesheet" type="text/css" href="mystyle.css"
               </code>
-            </ul>
```

Comment:
Review: You need to completely remove any unused html tag, or you make sure that every open tag is closed.
Reason: As part of Microverse best practice, the stray "</ul>" as seen in your code is not serving any purpose in your code. This can also cause linter error and will not make your code clean. I do not see a need for this tag in line 116, please remove that line.